addons/bus/models/bus.py

```diff
@@ -154,6 +157,23 @@ def _bus_last_id(self):
         return last.id if last else 0
 
 
+class BusSendableMixin(models.AbstractModel):
+    _name = 'bus.sendable'
```

Comment:
`bus.target` maybe?
`bus.listener`?
`bus.observer`?

And it should probably have `.mixin` suffix

Reply:
I went for `bus.listener.mixin` :+1: